lib/core/src/display/shape/text/text_field/content.rs

```diff
@@ -512,6 +573,28 @@ mod test {
         assert_eq!(two_lines_expected, two_lines_change.inserted_text_range());
     }
 
+    #[wasm_bindgen_test(async)]
+    async fn converting_location_to_offset() {
+        msdf_sys::initialized().await;
```

Comment:
Should this be needed to test conversions between (row,col) and (index) ?

Reply:
It always needed when you use fonts, and each creation of content require font